### ISSUE TITLE
A11y_OracleAssessmentForAzureDataStudio(ADS)_DatabaseMigrationAssessmentForOracle_DatabaseMigrationAssessmentForOracleDialog_Keyboard: When pressing Esc key instead of dismissing the tooltip whole dialog is getting dismissed.

### ISSUE THREAD

**Issue by v-akr** [microsoft/azuredatastudio#19583]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
 #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudio_Win32_Jun2021;#SQL Azure Data Studio;#Win11;#DesktopApp;#WCAG2.1.1;#Benchmark;#FTP;#Keyboard;

### Environment Details:
Application Name: Oracle Assessment for Azure Data Studio (ADS)
Operating System: Windows 11
Version :1.35.1

### Steps to Reproduce:

1. Download the azure data studio 
2. Open the application and tab to the Connections icon in left navigation.
3. Tab to the new connections button and press enter. Connection dialog box will open.
4. Fill Correct details in connection details and tab to the connect button and press enter. New connection will be added.
5. Tab to the Database Migration Assessment for Oracle button and then press enter. "Database Migration Assessment for Oracle" page will be opened.
6. Tab to the "Run New Assessment" button at top the page and press enter. New Database Migration Assessment for Oracle dialog box will open.
7. Tab till any tooltip and observe that When pressing Esc key instead of dismissing the tooltip whole dialog is getting dismissed.

### Actual:
When pressing Esc key instead of dismissing the tooltip whole dialog is getting dismissed.

### Expected:
When pressing Esc key tooltip should be dismissed not the whole dialog.

### User Impact:
Keyboard users will face difficulty because by pressing esc key tooltip is not getting dismissed as tooltip is hiding content.
